<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21::iface</class>
 <widget class="QWidget" name="s21::iface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="zoom_sub">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>%--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>605</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_rot_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>415</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_sub">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>201</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>x--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="z_rot_sub">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>440</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>z--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tops_colour">
   <property name="geometry">
    <rect>
     <x>877</x>
     <y>545</y>
     <width>145</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>выбрать цвет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="z_sub">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>261</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>z--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_colour">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>485</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать цвет фона</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_file_button">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>175</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Перемещение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lines_colour">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>545</y>
     <width>145</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>выбрать цвет</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>295</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштабирование</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>577</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Толщина:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>201</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>x++</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tops_size">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>575</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>575</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="line_type">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>599</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>сплошная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>пунктирная</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="z_rot_add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>440</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>z++</string>
   </property>
  </widget>
  <widget class="QComboBox" name="format">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>654</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>.jpeg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.bmp</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="y_sub">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>y--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>355</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вращение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_screen">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>655</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Сделать скриншот</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="z_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>265</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>605</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вид:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="z_rot_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>445</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="y_rot_add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>410</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>y++</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_gif">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>695</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Записать gif-анимацию преобразования</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_rot_add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>380</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>x++</string>
   </property>
  </widget>
  <widget class="QLabel" name="filepath_dots">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>941</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите файл.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tops_type">
   <property name="geometry">
    <rect>
     <x>906</x>
     <y>599</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>отсутствуют</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>квадратные</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>круглые</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="x_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>205</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="line_thickness">
   <property name="geometry">
    <rect>
     <x>803</x>
     <y>575</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="zoom_add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>320</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>%++</string>
   </property>
  </widget>
  <widget class="QPushButton" name="z_add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>261</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>z++</string>
   </property>
  </widget>
  <widget class="QPushButton" name="y_add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>230</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>y++</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>659</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>формат:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zoom_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>325</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="y_rot_sub">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>y--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>723</x>
     <y>525</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройка ребер:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_rot_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>385</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>525</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройка вершин:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>235</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x_rot_sub">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>x--</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
